app/Helpers/ContactHelper.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Helpers;
+
+use App\Models\User\User;
+
+class ContactHelper
+{
+    /**
+     * Get the name order that will be used when rendered the Add/Edit forms
+     * about contacts.
+     *
+     * @param Builder $query
+     * @param User $user
+     * @return string
+     */
+    public static function orderContactQueryByUserPreference(User $user, $query)
```

Comment:
```suggestion
     * @return Builder
     */
    public static function orderContactQueryByUserPreference(User $user, $query): Builder
```

Reply:
I couldn't get the builder return type to work for me. Looks like it's failing the build. Let me know if you want to me to make any other changes here to get this merged. Feel free to modify this branch as much as you'd like!